idea_plugin/build.gradle

```diff
@@ -15,31 +15,35 @@
  */
 
 plugins {
-  id "org.jetbrains.intellij" version "0.7.2"
+  id "org.jetbrains.intellij" version "1.2.1"
 }
 
 repositories {
   mavenCentral()
 }
 
 ext {
-  googleJavaFormatVersion = '1.10.0'
+  googleJavaFormatVersion = "1.12.0"
 }
 
-apply plugin: 'org.jetbrains.intellij'
-apply plugin: 'java'
+apply plugin: "org.jetbrains.intellij"
+apply plugin: "java"
+
+sourceCompatibility = JavaVersion.VERSION_11
+targetCompatibility = JavaVersion.VERSION_11
 
 intellij {
   pluginName = "google-java-format"
-  version = "211.6693.65-EAP-SNAPSHOT"
+  plugins = ["java"]
+  version = "2020.3"
```

Comment:
I like to build these against the latest version of IntelliJ possible. This causes a compile-time error if they added methods to `CodeStyleManager`, which they do from time to time (albeit not in a while now).